modules/services/mako.nix

```diff
@@ -33,6 +33,14 @@ in {
         '';
       };
 
+      maxHistory = mkOption {
+        default = 5;
```

Comment:
For future us, default is documented here: https://github.com/emersion/mako/blob/master/doc/mako.5.scd#global-configuration-options